### DISCUSSION TITLE
Blur filter gone after latest update to b164?

### DISCUSSION THREAD

**Discussion by Maxx2206** [Xaymar/obs-stream-effects#997] [Q&A] [answered]
Hi,

i was using the test version with OBS v28.
a few days a go i was prompted for a new version and installed it, now i realized my blur filters i used in some scenes do not work anymore.
in fact, if i "add" a new filter i fdont even find any blur option anymore.

is this intentional?

**Comment by skeletonbow** [comment, marked as answer]
StreamFX beta 164 only works with OBS 29.0.0beta2 release as mentioned in the beta release notes.

**Reply by Maxx2206** [reply]
oh, sorry, then...

to my excuse, i was prompted to update streamfx in obs v28 and in the prompt there was no mentioning that this is such a specific update.

rolled back to a151 anyway and it works...
